Sources/PointFree/Account/Change.swift

```diff
@@ -206,7 +206,15 @@ private let titleRowView = View<Stripe.Subscription> { subscription in
             ? ""
             : " Reactivate your subscription by submitting the form below."
           ]),
-        ])
+        ]
+        + (
+          subscription.discount?.coupon.valid == .some(true)
+            ? [
+                strong([text("Please note that changes to your subscription will remove your current discount.")])
```

Comment:
maybe we should put this in a `div` with some padding and a `Colors.yellow` background to pop a lil more?